### PR TITLE
Damage Now Scales with Medbay Hours

### PR DESCRIPTION
**Pull request by CMOisLing**
A robust medbay can pretty much stop anyone from dying and rarely do they ever have an issue dealing with injuries. This should spice things up a bit.

:cl:  
rscadd: Damage Now Scales with Medbay Hours
/:cl:

### CODE REVIEW

**Review by BurgerLUA — APPROVED** [review]

File: code/modules/jobs/job_types/medical_doctor.dm [hunk 1]
```diff
@@ -65,3 +65,14 @@
 	duffelbag = /obj/item/storage/backpack/duffelbag/med
 
 	chameleon_extras = /obj/item/gun/syringe
+
+var/most_hours_played = 0
```
Comment (Simplehorror) [inline code/modules/jobs/job_types/medical_doctor.dm:69]: nope

**Review by Mantlecrawler — APPROVED** [review]
10/10 Great PR 
LGTM

File: code/modules/mob/living/carbon/human/physiology.dm [hunk 2]
```diff
@@ -1,17 +1,17 @@
 //Stores several modifiers in a way that isn't cleared by changing species
 
 /datum/physiology
-	var/brute_mod = 1   	// % of brute damage taken from all sources
-	var/burn_mod = 1    	// % of burn damage taken from all sources
-	var/tox_mod = 1     	// % of toxin damage taken from all sources
-	var/oxy_mod = 1     	// % of oxygen damage taken from all sources
-	var/clone_mod = 1   	// % of clone damage taken from all sources
+	var/brute_mod = 1 + fun_mod   	// % of brute damage taken from all sources
```
Comment (Simplehorror) [inline code/modules/mob/living/carbon/human/physiology.dm:4]: doesnt work like that

**Review by Simplehorror — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/physiology.dm [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 	var/stamina_mod = 1 	// % of stamina damage taken from all sources
-	var/brain_mod = 1   	// % of brain damage taken from all sources
+	var/brain_mod = 1 + fun_mod   	// % of brain damage taken from all sources
 
-	var/pressure_mod = 1	// % of brute damage taken from low or high pressure (stacks with brute_mod)
-	var/heat_mod = 1    	// % of burn damage taken from heat (stacks with burn_mod)
-	var/cold_mod = 1    	// % of burn damage taken from cold (stacks with burn_mod)
+	var/pressure_mod = 1 + fun_mod	// % of brute damage taken from low or high pressure (stacks with brute_mod)
+	var/heat_mod = 1 + fun_mod    	// % of burn damage taken from heat (stacks with burn_mod)
+	var/cold_mod = 1 + fun_mod    	// % of burn damage taken from cold (stacks with burn_mod)
```
Comment (BurgerLUA) [inline code/modules/mob/living/carbon/human/physiology.dm:15]: wait hold on this code DOESN'T EVEN WORK

**Review by BurgerLUA — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/physiology.dm (hunk 3 above)
Comment (CMOisLing) [reply code/modules/mob/living/carbon/human/physiology.dm:15]: cry about it

Comment (Simplehorror) [reply code/modules/mob/living/carbon/human/physiology.dm:15]: stop copying my reviews

**Comment by CMOisLing** [comment]
Closing due to hateful DM's from Lizard mains.

**Comment by N3D6** [comment]
Reopen this what are you doing!!!!

**Comment by Necrosium** [comment]
Who cares bout lizards? Let them cope and seethe

**Pull request closed without merging**